generators/resources/mappings.v1.json.template

```diff
@@ -3,6 +3,7 @@
     {{#each credentialKeys}}
     "{{../serviceKey}}_{{this}}" : {
         "searchPatterns": [
+            "user-provided:{{../serviceName}}:{{access ../map this ../nestedJSON}}",
```

Comment:
i thought the pattern was `<devexId>_<serviceInstanceName>_<credKey>`

Reply:
@gconan Remember we are not using the tags since you mentioned that we won't have them, so we are only need the service names which is `serviceInfo.name` and the key to determine the credential value of the service. Does that work for you?